Apps/Sandcastle/gallery/Image-Based Lighting.html

```diff
@@ -32,6 +32,16 @@
 var environmentMapURL = '../../SampleData/EnvironmentMap/kiara_6_afternoon_2k_ibl.ktx';
 var modelURL = '../../SampleData/models/Spheres/MetalRoughSpheres.glb';
 
+// CesiumJS supports image-based lighting through the glTF extension `EXT_lights_image_based`.
```

Comment:
We don't actually support the extension yet. I was waiting until the extension was set before implementing it. @OmarShehata @emackey is the extension at a place where we could implement it and it won't change much?

Reply:
> is the extension at a place where we could implement it and it won't change much?

No, still under discussion.

Reply:
I don't think we should implement it until the proposed extension is at least merged, which it currently isn't. It seems to be converging though.

I think the fact that at least one program can export glTF's with this extension is promising, so that should help if we can get our hands on some exported models with it.